code/game/objects/items/plushes.dm

```diff
@@ -9,13 +9,88 @@
 	var/list/squeak_override //Weighted list; If you want your plush to have different squeak sounds use this
 	var/stuffed = TRUE //If the plushie has stuffing in it
 	var/obj/item/grenade/grenade //You can remove the stuffing from a plushie and add a grenade to it for *nefarious uses*
+	//--love ~<3--
+	gender = NEUTER
+	var/obj/item/toy/plush/lover
+	var/obj/item/toy/plush/partner
+	var/obj/item/toy/plush/plush_child
+	var/obj/item/toy/plush/paternal_parent	//who initiated creation
+	var/obj/item/toy/plush/maternal_parent	//who owns, see love()
+	var/list/scorned	//who the plush hates
+	var/list/scorned_by	//who hates the plush, to remove external references on Destroy()
+	var/heartbroken = FALSE
+	var/vowbroken = FALSE
+	var/young = FALSE
+	var/mood_message
+	var/list/love_message
+	var/list/heartbroken_message
+	var/list/vowbroken_message
+	var/list/parent_message
+	var/normal_desc
+	//--end of love :'(--
 
 /obj/item/toy/plush/Initialize()
 	. = ..()
 	AddComponent(/datum/component/squeak, squeak_override)
 
+	//have we decided if Pinocchio goes in the blue or pink aisle yet?
+	if(gender == NEUTER)
+		if(prob(50))
+			gender = FEMALE
+		else
+			gender = MALE
+
+	love_message		= list("\n\The [src] is so happy, \he could rip a seam!")
+	heartbroken_message	= list("\n\The [src] looks so sad.")
+	vowbroken_message	= list("\n\The [src] lost \his ring...")
+	parent_message		= list("\n\The [src] can't remember what sleep is.")
+
+	normal_desc = desc
+
 /obj/item/toy/plush/Destroy()
 	QDEL_NULL(grenade)
+
+	//inform next of kin and... acquaintances
+	if(partner)
+		partner.bad_news(src)
+		partner = null
+		lover = null
```

Comment:
you're nulling lover before trying to reference it below

Reply:
The else if?

Reply:
oh ignore me, didn't see the elseif